src/ReactWrapper.jsx

```diff
@@ -107,6 +108,26 @@ export default class ReactWrapper {
   }
 
   /**
+   * Makes a wrapper iterable, which is useful when using destructive
+   * assignment with `find()`
+   * @example
+   * const wrapper = shallow(<MyComponent />)
+   * const [fistLink, secondLink] = wrapper.find('a')
+   * @returns {Object}
+   */
+  [ITERATOR_SYMBOL]() {
+    let index = 0;
```

Comment:
@Aweary putting this at the end of the file? (and removing `export default` from the class definition)

```js
if (ITERATOR_SYMBOL) {
  ReactWrapper.prototype[ITERATOR_SYMBOL] = function() {
    return this.nodes[ITERATOR_SYMBOL]()
  }
}

export default ReactWrapper
```